search.py

```diff
@@ -61,10 +61,10 @@ def reindex_app(app):
     machine_class, tasks = SEARCHABLE_APPS[app]
     util.use_random_host('class-%s' % machine_class)
 
-    for task in tasks:
+    for t in tasks:
```

Comment:
I would rather use a more descriptive name that isn't task. At the moment I think it's easy for someone to assume there is no reason for it and change it back.

Alternatively you could change the import to just api and use 'api.task' elsewhere.